test/scripts/e2e_subs/e2e-app-simulate.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-date '+app-simple-test start %Y%m%d_%H%M%S'
```

Comment:
also line 101

Reply:
that one looks okay to keep since it's just a simple clear state app for the ABI contract